examples/fineFoods/admin/mui/src/pages/couriers/show.tsx

```diff
@@ -0,0 +1,198 @@
+                            {row.star}
+                        </Typography>
+                        <Rating
+                            name="rating"
+                            defaultValue={row.star}
+                            readOnly
+                        />
+                    </Stack>
+                ),
+            },
+        ],
+        [],
+    );
+
+    const { dataGridProps } = useDataGrid<IReview, HttpError>({
+        columns,
+        resource: "reviews",
+        initialSorter: [
+            {
+                field: "id",
+                order: "desc",
+            },
+        ],
+        permanentFilter: [
+            {
+                field: "order.courier.id",
+                operator: "eq",
+                value: courier?.id,
+            },
+        ],
+        initialPageSize: 4,
+        queryOptions: {
+            enabled: courier !== undefined,
+        },
+        syncWithLocation: false,
+    });
+
+    return (
+        <Grid container spacing={2}>
+            <Grid item xs={12} lg={3}>
+                <Paper sx={{ p: 2 }}>
+                    <Stack alignItems="center" spacing={1}>
+                        <Avatar
+                            src={courier?.avatar?.[0].url}
+                            sx={{ width: 120, height: 120 }}
+                        />
+                        <Typography variant="h6">
+                            {courier?.name} {courier?.surname}
+                        </Typography>
+                    </Stack>
+                    <br />
+                    <Stack spacing={1}>
+                        <CourierInfoText
+                            icon={<StoreOutlined />}
+                            text={courier?.store.title}
+                        />
+                        <CourierInfoText
+                            icon={<LocalPhoneOutlined />}
+                            text={courier?.gsm}
+                        />
+                        <CourierInfoText
+                            icon={<EmailOutlined />}
+                            text={courier?.email}
+                        />
+                        <CourierInfoText
+                            icon={<AccountBalanceOutlined />}
+                            text={courier?.accountNumber}
+                        />
+                        <CourierInfoText
+                            icon={<MapOutlined />}
+                            text={courier?.address}
+                        />
+                        <CourierInfoText
+                            icon={<DirectionsCarFilledOutlined />}
+                            text={courier?.licensePlate}
+                        />
+                    </Stack>
+                </Paper>
+            </Grid>
+            <Grid item xs={12} lg={9}>
+                <Stack direction="column" spacing={2}>
+                    <List cardHeaderProps={{ title: t("orders.orders") }}>
```

Comment:
```suggestion
                    <List cardHeaderProps={{ title: t("orders.orders") }}>
```
As in Ant design example we should hide `<CreateButton>`, https://github.com/pankod/refine/blob/master/examples/fineFoods/admin/src/pages/couriers/show.tsx#L112